app/controllers/organizations/dashboards_controller.rb

```diff
@@ -12,5 +12,24 @@ def show
     def current_organization
       @_current_organization ||= Organization.with_counts.find(params[:organization_id])
     end
+
+    def low_on_cards?
+      total_cards = @organization.total_payment_cards_count
+
+      card_limit = case
+                   when total_cards > 11_000
+                     2000
+                   when total_cards > 6000
+                     1000
+                   when total_cards > 0
+                     500
+                   else
+                     0 # TODO: what should happen when total cards are 0?
```

Comment:
I think you could just make the `else`  be a default of 500.

```suggestion                    
                   else
                     500
```